src/__tests__/add-delete.test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/extensions */
+
+/**
+ * @jest-environment jsdom
+ */
+
+import { expect } from '@jest/globals';
+import { addNewTaskToList, deleteItem } from '../update-to-do-list';
+
+jest.mock('../handle-storage.js');
+
+document.body.innerHTML = '<input type="text" placeholder="Add to your list..." id="new-task">';
+
+let toDoTasks = [];
+
+describe('My To-Do List', () => {
+  test('should add the new task into the list of to do list', () => {
+    const newTask = document.getElementById('new-task');
+    newTask.value = 'Wash the dishes';
+    toDoTasks = addNewTaskToList();
```

Comment:
- [ ] Please fix one test which is failing
![test](https://user-images.githubusercontent.com/44745855/131897803-792c94e7-72d5-48b7-842f-ac445b02f5bd.png)

- [ ] Consider testing positive and negative scenarios

Reply:
[Optional]
As suggested by previous TSE, it is always best practice to test your application for negative cases also.